docs/paper/dev/api/plugin-messaging.md

```diff
@@ -194,3 +195,29 @@ This is not a recommended way to ban players due to the fact that there may not
 but it is an example of how this can be used.
 
 :::
+
+#### MessageRaw
+
+The `MessageRaw` message type is used to send a raw chat component to a player. This is useful for sending clickable messages to players,
+or messages with hover events.
```

Comment:
Maybe also mention sending to players that aren't on the same backend server as an extra use case.